### DISCUSSION TITLE
Teleport to offline players.

### DISCUSSION THREAD

**Discussion by MCTGE** [Zrips/CMI#6524] [Feature Requests]
This is an awesome feature in Essentials that would be an amazing addition to CMI and that is the ability to teleport to offline players.  This makes investigating player reports so much easier, as well as resolving issues with broken chunks.

**Comment by RMHavelaar** [comment]
Omg yes, please!!!! This would be a game changer.

**Comment by CloudeLecaw** [comment]
U allready can tp to offline players since years now..and also can teleport offline players to you.

Use:
tpo [playerName] (playerName)
To tp to offline players.

And
tpohere [playerName] (playerName)
To tp others to you
